### PR TITLE
css-three-column-layout - Jonathan Matsuda

### PR DESCRIPTION
**Pull request by jonathanymatsuda**
Screen Shot 2021-10-10 at 11 40 54 PM

### CODE REVIEW

File: css-three-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      </div>
+      <div class="column-one-third text-column">
+        <h3>Integrates with existing tools</h3>
+        <p>Enzyme QMS integrates with your existing tools, such as JIRA, GitHub and Jenkins with Enzyme</p>
+      </div>
+      <div class="column-one-third text-column">
+        <h3>In-depth Onboarding</h3>
+        <p>We train your entire team and provide you with support when and as it's needed</p>
+      </div>
+    </div>
```
Comment (codingcodymiller) [inline css-three-column-layout/index.html:41]: I can understand why you would do this as related to layout classes, but this sort of HTML setup makes it look as though the image is separate content from the text for each column, and that is not an accurate representation of the content. The icon, heading, and descriptive text should all be grouped together in one `column` class. Remember, the goal of HTML is to organize your elements in a way that semantically describes the content accurately. If elements are related, they should be grouped together so it's clear through the HTML that those items are related content.

File: css-three-column-layout/index.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      </div>
+      <div class="column-one-third">
+        <h3 class="letter-spacing text-align-center">Integrates with existing tools</h3>
+        <p class="text-align-center">Enzyme QMS integrates with your existing tools, such as JIRA, GitHub and Jenkins with Enzyme</p>
+      </div>
+      <div class="column-one-third">
+        <h3 class="letter-spacing text-align-center">In-depth Onboarding</h3>
+        <p class="text-align-center">We train your entire team and provide you with support when and as it's needed</p>
+      </div>
+    </div>
```
Comment (codingcodymiller) [inline css-three-column-layout/index.html:41]: This is still overly separated. The image and text are related, so they should all be grouped together. The image, heading, and text are all related, so they should all be children of one column class

Comment (jonathanymatsuda) [reply css-three-column-layout/index.html]: Screen Shot 2021-10-14 at 5 37 59 PM

**Review by codingcodymiller — APPROVED** [review]
Nice work on this!! I see a lot of `em` though, be cautious with that unit in the future. It can be helpful, but without proper planning, it can lead to a lot of unintended side effects.

**Pull request merged**